### DISCUSSION TITLE
Achievements view misaligned on iPadOS

### DISCUSSION THREAD

**Discussion by Jerry23011** [community/community#44950] [Mobile] [answered]
### Select Topic Area

Bug

### Body

GitHub achievements and status on iPadOS are not properly aligned. Please see screenshots below.
- landscape

![IMG_4282](https://user-images.githubusercontent.com/89069957/213843319-d84d8209-3aa3-427b-a18d-a66e9273a61f.png)
- portrait

![IMG_4283](https://user-images.githubusercontent.com/89069957/213843334-cc2125a6-99c8-4b59-82b1-5fc3afb2ccef.png)
- sidecar

![IMG_4285](https://user-images.githubusercontent.com/89069957/213843350-171bd00d-b20c-4129-94b5-0985c3e050c1.png)

iPadOS 16.2
iPad Air 3rd 10.5'

**Comment by jge162** [comment]
yeah its a bit off, good eye

**Comment by iskae** [comment, marked as answer]
Hi @Jerry23011 👋 

Thanks so much for your bug report ❤️ I have created an internal issue for this and hopefully we can fix this asap 🙇 

Please stay tuned for the future versions for the fix 🎉

**Reply by taki-on** [reply]
Fix is on the way :) It will go out public second next week.
